tests/integration/targets/ios_logging_global/tests/cli/rendered.yaml

```diff
@@ -52,8 +52,6 @@
             tag: hostname
           persistent:
             batch: 4444
-          policy_firewall:
```

Comment:
Are these attributes deleted?

Reply:
so with the image version upgrade, specific to the l3 image, this is an invalid attribute.
We might want to deprecate the attribute if it's not valid for l2 too, we would need to wait before we have an image with us.

Reply:
Okay, got it.